setup.py

```diff
@@ -48,6 +48,7 @@
         'console_scripts': [
             # 'command = some.module:some_function',
         ],
+        "intake.catalogs": ["local = sirepo_bluesky:sirepo_bluesky_catalog_instance"],
```

Comment:
That's an interesting approach. I haven't encountered this way of setting the catalogs. Was it based on any intake example? If so, could you please provide a link?

Reply:
These are the [instructions](https://blueskyproject.io/databroker/reference/configuration.html?highlight=entry%20point#advanced-configuration-via-python-package) we followed (but only after I sent @jennmald down a couple wrong paths).

Reply:
Thanks! That's something I haven't noticed in the `databroker` documentation.

Reply:
I think it would make sense to put that link as a comment next to the `"intake.catalogs"` line. That will help newcomers to get onboard faster.